src/pages/UploadPage.tsx

```diff
@@ -70,16 +123,17 @@ export default function UploadPage() {
           component="label"
           sx={{ width: '100px' }}
           onClick={sendFiles}
+          disabled={isSendButtonDisabled}  // Disable the button conditionally
         >
           Send
         </Button>
       </Box>
       {alert ? (
-        <Alert severity="error">Oops, something went wrong... Please add files</Alert>
+        <Alert severity="error">Oops, something went wrong... Please add files and select a vacancy</Alert>
```

Comment:
Since you cannot click on the send button without selecting a vacancy or having a file uploaded, change text to something like 'Oops, something went wrong!'.